UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Services/FssServiceTests.cs

```diff
@@ -222,14 +221,13 @@ public async Task DoesDownloadFile_Returns_DownloadPath_If_ValidRequest()
                     },
                 });
 
-            Stream? result = await _fssService.DownloadFile("M01X02", "/batch/621e8d6f-9950-4ba6-bfb4-92415369aaee/files/M01X02.zip");
-
-            Assert.That(result, Is.Not.Null);
+            bool result = await _fssService.DownloadFile("M01X02", "/batch/621e8d6f-9950-4ba6-bfb4-92415369aaee/files/M01X02.zip", 10000, @"D:\POS");
 
+            Assert.That(result, Is.True);
```

Comment:
why is there 2 asserts in a unit test, these need to focus on assertion of just a single point

Reply:
As long as the asserts are in a block of asserts this we think is acceptable. Therefore it must always be:

- Arrange
- Action
- Assert

So never do an action followed by asserts AND then do another action (e.g. see test Does_GetFileMD5_Returns_FileDetails_With_Hash which breaks this rule)